floris/simulation/wake_velocity/jensen.py

```diff
@@ -107,7 +113,6 @@ def function(
         dz = ne.evaluate("z - z_i")
 
         we = self.we
-        NUM_EPS = JensenVelocityDeficit.NUM_EPS
```

Comment:
Are you suggesting to add `NUM_EPS` to the Jensen class? In other words, replace the left hand side or right hand side with `self.NUM_EPS`?

Reply:
The right-hand side, so `NUM_EPS = self.NUM_EPS`, sorry for the confusion.

Reply:
Ah, thanks for clarifying. I included it as a module-level global parameter in all of the wake models because in the Gauss Deflection model it's used outside of the model class (see [here](https://github.com/rafmudaf/floris/blob/attrs_fix/floris/simulation/wake_deflection/gauss.py#L318)). These functions at the bottom of the Gauss deflection model module can also be used for any other wake model. All the wake models should use it in a consistent way, so unfortunately the Gauss model is the driver for this architecture.

FWIW I agree that it would be organizationally better to get `NUM_EPS` from `self`, so if you see a way to navigate the GCH functions please let me know.

Reply:
I see, I was really only referring to this specific instance because the class' name is `JensenVelocityDeficit`, so this is seemed more like a long way around `self.NUM_EPS`. This is by no means a deal breaker for me, just something that caught my eye.

Reply:
Ok thanks for catching it and pointing it out. I agree with you and I'll continue to think about it, but for this pull request I'd like to hold off making this change.